src/routes/HomePage.jsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { filterCoin, sortByPrice, sortByRank } from '../redux/coins/coinsSlice';
+import rightArrow from '../styles/arrow_circle_right.png';
+
+const HomePage = () => {
+  const { coinsList } = useSelector((store) => store.coins);
+  const dispatch = useDispatch();
+  const itemsPerPage = 10;
+  const [currentPage, setCurrentPage] = useState(1);
+  const indexOfLastItem = currentPage * itemsPerPage;
+  const indexOfFirstItem = indexOfLastItem - itemsPerPage;
+  const currentItems = coinsList.slice(indexOfFirstItem, indexOfLastItem);
+  const totalPages = Math.ceil(coinsList.length / itemsPerPage);
+
+  const handlePageChange = (pageNumber) => {
+    setCurrentPage(pageNumber);
+  };
+
+  const orderCoins = (order) => {
+    if (order === 'rank') {
+      dispatch(sortByRank());
+    } else if (order === 'priceUsd') {
+      dispatch(sortByPrice());
+    }
+  };
+
+  return (
+    <div className="home-page-wrapper">
+      <main>
+        <ul className="grid-container">
+          {currentItems.map((coin) => (
+            <li key={coin.id} className="li-home-page">
```

Comment:
- [ ] As requested by the previous reviewer 👇🏻 

![image](https://github.com/Sparktan24/metrics-webapp-cryptocurrency/assets/79058364/75a86cc6-98cd-4eaf-802e-1ce2cce4b5f2)


- [ ] Style your components to match the design provided. Make sure to add this intro top section and right navigation icons 👇🏻 


![image](https://github.com/Sparktan24/metrics-webapp-cryptocurrency/assets/79058364/1cf4467a-8638-42e9-8e79-593f5bfaf2ad)